src/main/java/com/revature/studyforce/flashcard/controller/QuizController.java

```diff
@@ -0,0 +1,49 @@
+package com.revature.studyforce.flashcard.controller;
+
+import com.revature.studyforce.flashcard.dto.NewQuizDTO;
+import com.revature.studyforce.flashcard.dto.QuizDTO;
+import com.revature.studyforce.flashcard.dto.UpdateQuizDTO;
+import com.revature.studyforce.flashcard.model.Quiz;
+import com.revature.studyforce.flashcard.service.QuizService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin
+@RequestMapping("/flashcards/quiz")
+public class QuizController {
```

Comment:
No documentation included in any of the methods. Include parameters, returns expected, and @links to service layer methods you are using.
EX: {@link BatchService#getAllBatches(int, int, String, String)}